pkg/plugin/standalone/v1/client.go

```diff
@@ -154,6 +156,9 @@ func unwrapGRPCError(err error) error {
 	if !ok {
 		return err
 	}
+	if st.Code() == codes.Unimplemented {
+		return cerrors.Errorf("%s: %w", st.Message(), cplugin.ErrUnimplemented)
```

Comment:
This ensures backward compatibility with older plugins whenever we introduce methods in the future. We can check if a plugin call returns `plugin.ErrUnimplemented` and know that the method does not exist. Note that this can't happen in a built-in plugin because Conduit forces all built-in plugins to use the same protocol and SDK version (latest).

I did not create a test for this as it's relatively hard to do, but I tested it out manually with an older plugin and ensured it works.